### PR TITLE
Cosmo to/from table row

### PR DESCRIPTION
**Pull request by nstarman**
### Description

Register "astropy.table.Row" with Cosmology I/O.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Do the proposed changes actually accomplish desired goals?
- [x] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [x] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [x] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [x] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [x] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [x] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [x] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [x] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

**Review by mhvk — APPROVED** [review]
Apart from a small copy & paste typo and a very small request for clarification in the docstring, this all looks good, so I'll approve.

File: astropy/cosmology/io/row.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        `~astropy.io.registry.UnifiedReadWriteMethod`
+    cosmology_in_meta : bool
+        Whether to put the cosmology class in the Table metadata (if `True`) or
+        as the first column (if `False`, default).
+
+    Returns
+    -------
+    `~astropy.table.Row`
+        With columns for the cosmology parameters, and metadata and
+        cosmology class name in the Table's ``meta`` attribute
```
[the remark this answers is not in the document]
Comment (nstarman) [reply astropy/cosmology/io/row.py]: Yes. Thanks for the catch!

File: astropy/cosmology/io/row.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    mapping["name"] = name
+    mapping.setdefault("cosmology", meta.pop("cosmology", None))
+    mapping["meta"] = meta
+
+    # build cosmology from map
+    return from_mapping(mapping, move_to_meta=move_to_meta, cosmology=cosmology)
+
+
+def to_row(cosmology, *args, cosmology_in_meta=False):
+    """Serialize the cosmology into a `~astropy.table.QTable`.
```
[the remark this answers is not in the document]
Comment (nstarman) [reply astropy/cosmology/io/row.py]: Thanks!

**Comment by nstarman** [comment]
Failures appear unrelated, so merging. 
Thanks @mhvk!

**Pull request merged**